admin/allow-services.rst

```diff
@@ -38,36 +38,42 @@ Configure proxy settings
 
 If you need to use a proxy, set one of the following environment variables according to your needs:
 
-- ``HTTP_PROXY``: The HTTP proxy address
-- ``HTTPS_PROXY``: The HTTPS proxy address
-- ``NO_PROXY``: If a proxy is defined, sets addressess that don't use the proxy
+- ``HTTP_PROXY``: Address of the proxy for HTTP request. Port is optional.
+- ``HTTPS_PROXY``: Address of the proxy for HTTPS request. Port is optional.
+- ``NO_PROXY``: If a proxy is defined, sets addressess that don't use the proxy.
 
-The following examples show how to set the ``HTTPS_PROXY`` environment variable for hosts and containers:
+The following examples show how to set the ``HTTP_PROXY`` and ``HTTPS_PROXY`` environment variable for hosts and containers:
 
 .. tabs::
 
    .. code-tab:: powershell Windows
 
+      $Env:HTTP_PROXY = "proxy.address:8080"
       $Env:HTTPS_PROXY = "proxy.address:443"
 
    .. code-tab:: bash Linux
 
+      export HTTPS_PROXY = "proxy.address:8080"
```

Comment:
Same as before
normally the same proxy address is used for both, HTTP_PROXY and HTTPS_PROXY, so I suggest to stick to the same port to avoid confusion